lib/utils/general_utils.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'set'
```

Comment:
set is only used in package utils, so this can go
```suggestion
```

Reply:
Thanks!!